src/component/common/Table/TableSearch/TableSearch.tsx

```diff
@@ -16,14 +18,37 @@ interface ITableSearchProps {
 export const TableSearch: FC<ITableSearchProps> = ({
     initialValue,
     onChange = () => {},
-    placeholder = 'Search',
+    placeholder: customPlaceholder,
 }) => {
     const [searchInputState, setSearchInputState] = useState(initialValue);
     const [isSearchExpanded, setIsSearchExpanded] = useState(
         Boolean(initialValue)
     );
     const [isAnimating, setIsAnimating] = useState(false);
+    const [isTooltipOpen, setIsTooltipOpen] = useState(false);
     const debouncedOnSearch = useAsyncDebounce(onChange, 200);
+    useKeyboardShortcut({ modifiers: ['ctrl'], key: 'f' }, () => {
+        setIsTooltipOpen(true);
+    });
+    useKeyboardShortcut(
+        { modifiers: ['ctrl'], key: 'k', preventDefault: true },
+        () => {
+            setIsSearchExpanded(true);
+        }
+    );
+    useKeyboardShortcut({ key: 'Escape' }, () => {
```

Comment:
Maybe it could be interesting to also use the same shortcut to close the search? Like a toggle? So close with either `escape` or `ctrl + K` again.

Reply:
Seems like we now have this behaviour:
 - No matter whether we're focused on the field, `escape` toggles the dropdown. If we're focused, it doesn't blur;
 - ⌘+K works well to focus the field, but no longer acts like a toggle - If we're focused, it doesn't blur;